src/base-components/Listing/BaseListing.js

```diff
@@ -113,14 +121,26 @@ export default {
     created: function() {
         if (this.data != null){
             this.populateCurrentStateAndData(this.data);
+            this.setParamsFromUrl(false); // set filters&ordering from url (pagination is updated in populateCurrentStateAndData)
         } else {
+            this.setParamsFromUrl();
             this.loadData();
         }
 
         var _this = this;
         setInterval(function(){
             _this.now = moment().tz(_this.timezone).format('YYYY-MM-DD HH:mm:ss');
         }, 1000);
+
+        this.updateFiltersInChildComponents(this.filters);
+
+        window.onpopstate = function(event) {
+            if(_this.dynamicUrl) {
```

Comment:
Nemam rad tieto prebindovavacky this na _this, ale ked asi sa ti to nechce opravovat vsade, tak to nechaj kludne

Reply:
cele craftable je pisene tymto stylom, tu som len pokracoval v zadanom trende :) tiez v tom nevidim zmysel, ale tak potom by sme to mali cele refaktorovat